pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.9</version>
+        <version>3.4</version>
```

Comment:
I have ever heard that 3.x officially only supports core 2.x.
It might be better to use plugin-pom 2.x.
https://github.com/jenkinsci/copyartifact-plugin/pull/97#discussion_r155136724
(on the other hand, I'm not sure what "only supporting 2.x" exactly means)

Reply:
OK, I'm happy to hear that.
I'll use plugin-pom 3.x also in other plugins.